README.rst

```diff
@@ -38,13 +38,14 @@ also you can install VmShepherd manually using a make:
 
   make install
 
-Also we provide a dockerfile which can be used for a development installation:
+There is also a third way with using a docker container:
 
 ::
 
   docker build -t vmshepherd .
-  docker run -it  -p 8888:8888 -p 8000:8000 vmshepherd run
+  docker run -d --name vmshepherd -v /etc/vmshepherd/confs/:/etc/vmshepherd -p 8888:8888 vmshepherd -c /etc/vmshepherd/settings.yaml
```

Comment:
what about port `8000`?

Reply:
is for a sphinx, well i removed that, and leave pure installation of vmshepherd